src/typeDefs/inputs.ts

```diff
@@ -325,7 +325,7 @@ export const inputs = gql`
   input PostUpdateInput {
     text: String
     title: String
-    imageUrl: String
-    videoUrl: String
+    imageUrl: URL
```

Comment:
This implementation is probably flawed btw whereever it's used. If the client wants to update their post they need to send back a new image or video, not a url. There are many other places too where this way of updating objects is used.

Reply:
I was thinking the same. But currently I do not think this implementation is functional, both in admin as well as in talawa. 
So I think its better we leave it as a URL, till we can standardise their implementations across the projects.

Reply:
Ok